libs/components/pages/src/lib/modules/action-hub/action-hub.module.ts

```diff
@@ -17,6 +17,7 @@ import { SkyActionHubComponent } from './action-hub.component';
 
```

Comment:
What do you think about making the action hub use this new component so that any new features we add to the recent links component get applied to the action hub? This would also ensure that both setups share the same features going forward.

Reply:
I've swapped in this component. I'm reluctant to completely refactor action hub to use page layout in this PR, but that's doable.